introduction.md

```diff
@@ -1,4 +1,3 @@
-hello , everyone , i'm a newbie from china also called as tianchao, 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?